inspec.yml

```diff
@@ -229,3 +229,8 @@ inputs:
     desc: "The minimum Postgres version allowed by the organization"
     type: string
     value: ""
+  
+  - name: priv_esc
+    desc: "Privilege elevation must be used only where necessary and protected from misuse"
+    type: string
+    value: "|t"
```

Comment:
And then here it should look at the user specifically instead of the fact that something came back with priveledged escalation (meaning "t" for true in priv escalation)